index.html

```diff
@@ -686,6 +686,51 @@ <h4>Redirected and Proxied External Content</h4>
           </p>
         </section>
       </section>
+
+      <section id="state-tokens">
+        <h3>State Tokens</h3>
+        <blockquote class="informative">
+          Non-normative note: State tokens are similar to ETags, providing a mechanism for clients to detect changes
+          in resource state without having to compare the entire entity body, and to perform conditional updates.
+          Like ETags, the principle behind state tokens is that implementations have the best information on
+          resource state, and can choose the most accurate and efficient mechanism for clients to detect changes.
+          State tokens differ from ETags in that there is no expectation that the state token must change if
+          the resource is not byte-for-byte identical, when the underlying state of the resource has not changed.
```

Comment:
Sorry for not noticing this earlier... but ETags are for a _representation_, or _entity body_ as we have been saying, rather than resources. I think fix is easy (2 places). Here: "...**the entity body** is not byte-for-byte identical"

Reply:
Good catch @zimeon — I'll update the PR.